core/src/main/java/io/questdb/std/str/AbstractCharSequence.java

```diff
@@ -57,6 +57,14 @@ public final CharSequence subSequence(int start, int end) {
         if (start == 0 && end == length()) {
             return this;
         }
+        // it's tempting apply this optimization:
```

Comment:
i don't think musings in comments are useful long term and should be removed

Reply:
👌